res/values/strings.xml

```diff
@@ -63,7 +63,6 @@
     <string name="next">Next</string>
     <string name="error">Error</string>
     <string name="error_x">Error: %1$s</string>
-    <string name="error_no_network">Network unavailable.</string>
```

Comment:
Is it OK to just remove this string, or will there be problems as other UIs depend on it? (same below)

Reply:
`error_no_network` is not used on iOS

Reply:
Not used on Desktop, either as far as I can see (https://github.com/deltachat/deltachat-desktop/search?p=1&q=error_no_network)

Reply:
i once wrote a script to answer these kind of questions :)

```
./scripts/grep-string.sh error_no_network
```

shows usages only in node's CHANGELOG.md, so, yes, the string can be removed :)